packages/maps/src/components/result/ReactiveMap.js

```diff
@@ -865,6 +880,7 @@ const mapStateToProps = (state, props) => ({
 			&& state.selectedValues[`${props.componentId}-page`].value - 1)
 		|| 0,
 	time: (state.hits[props.componentId] && state.hits[props.componentId].time) || 0,
+	error: state.error[props.componentId],
```

Comment:
We should have setError from `actions` to set Errors in case GoogleMap / OpenStreetMap throws any error.

Reply:
@lakhansamani Is there any way to catch errors from GoogleMap / OpenStreetMap?
i.e are we already doing it somewhere?

Reply:
For google map one check we can add is

https://github.com/appbaseio/reactivesearch/pull/930/files#diff-336a5c69e66e07ecdcb03bdcb94c0868R272

Reply:
Actually, that check is it to avoid error in case of Network failure, if there is no connection then the maps script won't be loaded and `window.google` will be absent.
Before, It was crashing the app.
In case of network failure, _msearch will also fail so `renderError` will work.
Although we can handle maps error in `GeoDistanceDropdown` but in `ReactiveMap` there is no use case for now.